ob-pattern-1/values.yaml

```diff
@@ -138,6 +138,10 @@ wso2:
       gateway:
         hostname: wso2-obam-gateway
 
+nfs:
```

Comment:
@kdanW I think we should provide support for an internal deployment of the NFS server (which you have already done) as well as allow a user to plug an external NFS server to the deployment. Let us handle this via an internal discussion.

Reply:
I will send this change in a separate PR